ConcernsCaseWork/Service.Redis/Teams/TeamsCachedService.cs

```diff
@@ -24,7 +22,7 @@ public TeamsCachedService(ILogger<TeamsCachedService> logger, ITeamsService team
 		}
 
 		public Task<ConcernsCaseworkTeamDto> GetTeam(string ownerId)
```

Comment:
It's an interesting pattern you're using here - returning a task rather than executing it. Not one I've seen before. Would be good to know the background of it. I had to pull the branch to follow the code through to see that the code does actually get executed at some layer. Would be interesting to see if cancellation tokens are still used in this pattern too.

Reply:
Cancellation tokens are still used. It allows fail-fast. More driven by how autofixture likes things in the idioms.